pipeline/metadata/flatten_satellite.py

```diff
@@ -38,20 +38,34 @@
 
 SATELLITE_BLOCKPAGES_FILE = 'blockpages.json'  # v2.2
 
-SATELLITE_FILES = [
-    SATELLITE_RESOLVERS_FILE,
-    SATELLITE_RESULTS_FILE,
-    SATELLITE_TAGGED_ANSWERS_FILE,
-    SATELLITE_ANSWERS_CONTROL_FILE,
-    SATELLITE_INTERFERENCE_FILE,
-    SATELLITE_INTERFERENCE_ERR_FILE,
-    SATELLITE_ANSWERS_ERR_FILE,
-    SATELLITE_TAGGED_RESOLVERS_FILE,
-    SATELLITE_RESPONSES_CONTROL_FILE,
-    SATELLITE_TAGGED_RESPONSES,
-    SATELLITE_BLOCKPAGES_FILE,
+# Files containing metadata for satellite DNS resolvers
+SATELLITE_RESOLVER_TAG_FILES = [
+    SATELLITE_RESOLVERS_FILE, SATELLITE_TAGGED_RESOLVERS_FILE
 ]
 
+# Files containing metadata for satellite receives answer ips
+SATELLITE_ANSWER_TAG_FILES = [
+    SATELLITE_TAGGED_ANSWERS_FILE, SATELLITE_TAGGED_RESPONSES
+]
+
+# Files containing satellite ip metadata
+SATELLITE_TAG_FILES = SATELLITE_RESOLVER_TAG_FILES + SATELLITE_ANSWER_TAG_FILES
+
+# Files containing satellite blockpage information
+SATELLITE_BLOCKPAGE_FILES = [SATELLITE_BLOCKPAGES_FILE]
```

Comment:
This is one file. No need for the `_FILES` constant.

Reply:
removed